view/partial/remember-patient-pin.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="modal" id="remember-patient-dialog" role="dialog">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal">
+                  <span>
+                      &times;
+                  </span>
+                  <span class="sr-only">
+                      Close
+                  </span>
+              </button>
+              <h4 class="modal-title">
+                  Patient {{ patient.pin }} Created!
+              </h4>
+            </div>
+            <div class="modal-body">
+              <p>
+                  Please copy down the PIN {{ patient.pin}}. This information is very important.
```

Comment:
nit, have a space before and after double curlies. e.g. `{{ patient.pin }}`.
Also there is a merge conflict that needs to be resolved.